com.arc.embeddedcdt/src/com/arc/embeddedcdt/LaunchConfigurationConstants.java

```diff
@@ -70,4 +70,8 @@
     // Constants
     static final String ASHLING_DEFAULT_PATH_WINDOWS = "C:\\AshlingOpellaXDforARC\\ash-arc-gdb-server.exe";
     static final String ASHLING_DEFAULT_PATH_LINUX = "/usr/bin/ash-arc-gdb-server";
+
+    //Custom Gdbserver
+    String ATTR_DEBUGGER_CUSTOM_GDBSERVER_BIN_PATH =  LAUNCH_ID + ".debugger_custom_gdbsever_bin_path"; //$NON-NLS-1$
```

Comment:
NON-NLS is either needed on both lines or on neither. Also there should be a space after `//`.